widgets/src/components/ProcessForm/ProcessFormContainer.js

```diff
@@ -78,23 +80,54 @@ class ProcessFormContainer extends React.Component {
     return null;
   }
 
+  submitProcessForm(form) {
+    this.setState({ submitting: true }, async () => {
+      const { config } = this.state;
+      const { onSubmitForm } = this.props;
+
+      let response = { errors: ['messages.errors.dataSaving'] };
```

Comment:
what's the idea behind this?

Reply:
i just place a default response to this in case there are no response. I guess I have to restructure this for a better look :p